src/main/scala/com/research/nomad/markii/dataflow/AFTDomain.scala

```diff
@@ -423,6 +423,36 @@ case class AFTDomain(private val localNodeMap: Map[Local, AccessPath[AbsValSet[A
       d
     }
   }
+  def setDialogTitle(ctxMethod: SootMethod, stmt: Stmt, viewLocal: Local, param: Value): AFTDomain = {
+    var d = copy()
+    d = d.updateLocalViewNodes(ctxMethod, stmt, viewLocal,{
+      case viewNode: ViewNode =>
+        val newAttr = Set((AndroidView.ViewAttr.dialogTitle, param)).collect {
+          case (attr, stringConstant: StringConstant) =>
+            (attr, stringConstant.toString)
+        }
+        val attr = viewNode.getAttrs ++ newAttr
+        viewNode.setAttributes(attr.toSet)
+      case n => n
+    })
+    d
+  }
+
+  def setDialogMessage(ctxMethod: SootMethod, stmt: Stmt, viewLocal: Local, param: Value): AFTDomain = {
+    var d = copy()
+    d = d.updateLocalViewNodes(ctxMethod, stmt, viewLocal,{
+      case viewNode: ViewNode =>
+        val newAttr = Set((AndroidView.ViewAttr.dialogMessage, param)).collect {
+          case (attr, stringConstant: StringConstant) =>
+            (attr, stringConstant.toString)
```

Comment:
Same. I think you should use `stringConstant.value`.

I suspect that the quote and unquote in the fact is related to this.